.github/workflows/main.yml

```diff
@@ -16,15 +14,13 @@ jobs:
       fail-fast: false
       matrix:
         scala:  [2.12.19, 2.13.13]
-        db-version: [3.11.17, 4.0.12, 4.1.4, 5.0-beta1, dse-6.8.44]
+        db-version: [3.11.19, 4.0.17, 4.1.8, 5.0.4, dse-6.8.44]
 
     steps:
       - uses: actions/checkout@v4
 
-      - name: ccm pip installation
-        uses: BSFishy/pip-action@v1
-        with:
-          packages: git+https://github.com/riptano/ccm.git@d74db63d75112908a77b6c80757df9343fdc3338
+      - name: Install ccm via pip
+        run: pip install git+https://github.com/apache/cassandra-ccm.git@trunk
```

Comment:
Is there a risk of pulling directly from Trunk breaking something if trunk has some breaking change? Looks like there aren't any tags in the repo to use but maybe tie to a SHA rather than trunk itself? Or maybe we can start tagging CCM?

Reply:
there's the `cassandra-test` tag that's mean for CI.

but wouldn't we rather catch failures (even) faster ?  it's not like those making changes to cassandra-ccm will have this repo in mind… (it's usually just cassandra-dtest that's on folks' radar)

Reply:
(Note I have had the same conversation on the Analytics side, and my plan was to try to put in a nightly build that runs against C* _trunk_ to specifically test for and catch incompatibilities faster, but we otherwise test against released versions)

Reply:
Ideally it would be best to test against both the last stable version of CCM as well as its trunk.  But there are no versions of CCM (currently), and the QA stability benefit of `cassandra-test` over `trunk` is little.

Failures would be a CCM breakage, to be caught early and reported back to CCM.

A separate reason/question between the `cassandra-test` vs `trunk` choice is atomicity of a CI run.  The main benefit I've seen of using the `cassandra-test` tag is to ensure it's only changed in between CI runs, so we don't get mixed results.  (We don't expect ccm devs to be holding back commits conscious of what CI is doing.)  While I doubt anyone will be checking GHA runs from this repo before re-tagging `cassandra-test`, it might still have some benefit 🤷 

what's the verdict ?

Reply:
done.